lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -184,6 +184,33 @@ def _onInstanceableChanged(self, value):
         with mayaUsdLib.UsdUndoBlock():
             self.prim.SetInstanceable(value)
 
+# Custom control for all array attribute.
+class ArrayCustomControl(aecustom.CustomControl):
```

Comment:
I think maybe we shouldn't derive from this CustomControl anymore. It is meant for Maya custom controls as the base class tries to get a plug name (which UFE items don't have). I have a local change to remove this from the other custom control in this file.

Just make your class derive from 'object', remove the args/kwargs from the __init__ and change buildControlUI to (you also need a OnReplace).

```
def onCreate(self, *args):
...
def onReplace(self, *args):
    pass
```